app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -120,16 +120,22 @@ def miq_credential_types
   def credentials
     provider_module = ManageIQ::Providers::Inflector.provider_module(collector.manager.class).name
     collector.credentials.each do |credential|
+      credential_kind = credential.kind
```

Comment:
I don't quite see what is effectively changed in this file

Reply:
leftovers of previous approach